bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cf-deployment-tracker-service",
+  "version": "1.0.0",
+  "private": false,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components"
```

Comment:
See previous note about moving `bower_components` into `public` (and likely renaming it).